### PR TITLE
Feat: Add test for valid student submission (#34)

### LINKED ISSUE #34: `execute`: If student submission, result is valid (ex02)
If a student submits, the action should produce a valid JSON result (not throw exception).

### PR DESCRIPTION
**Pull request by markusnewtonh**
Add unit test to validate student's submission in `execute()`. 
Fix: re-add previously removed test `testAccessControl_feedbackQuestionDoesNotExist_shouldThrowEntityNotFoundException()`.

Fixes #34

### CODE REVIEW

File: src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java [hunk 1]
```diff
@@ -156,6 +201,19 @@ public void testAccessControl_studentSubmissionPastEndTime_shouldAllowIfBeforeDe
         verifyCanAccess(submissionParams);
     }
 
+    @Test
```
Comment (markusnewtonh) [inline src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java:204]: This test was accidentally removed in another merge of a PR. Re-added it after a discussion with @ElHachem02.

File: src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java [hunk 2]
```diff
@@ -156,6 +201,19 @@ public void testAccessControl_studentSubmissionPastEndTime_shouldAllowIfBeforeDe
...
+    public void testAccessControl_feedbackQuestionDoesNotExist_shouldThrowEntityNotFoundException() throws Exception {
```
Comment (markusnewtonh) [inline src/test/java/teammates/ui/webapi/SubmitFeedbackResponsesActionTest.java:205]: This test was accidentally removed in another merge of a PR. Re-added it after a discussion with @ElHachem02.

**Comment by markusnewtonh** [comment]
Will update this PR after #51 is resolved.

**Review by FalkWasTaken — APPROVED** [review]
Looks good!

**Pull request merged**